internal/scaffold/controller_kind.go

```diff
@@ -243,17 +248,36 @@ func (r *Reconcile{{ .Resource.Kind }}) Reconcile(request reconcile.Request) (re
 		reqLogger.Info("Creating a new Pod", "Pod.Namespace", pod.Namespace, "Pod.Name", pod.Name)
 		err = r.client.Create(context.TODO(), pod)
 		if err != nil {
+			if err := r.setStatusNotReady(context.TODO(), instance, err.Error()); err != nil {
+				return reconcile.Result{}, err
+			}
 			return reconcile.Result{}, err
 		}
 
-		// Pod created successfully - don't requeue
-		return reconcile.Result{}, nil
+		// Pod created successfully - requeue
+		return reconcile.Result{Requeue: true}, nil
```

Comment:
Why is required to Requeue here?

Reply:
@camilamacedo86 I just reverted this code to simplify the PR.